docs/api/browser-window.md

```diff
@@ -205,6 +205,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
     `#FFF` (white).
   * `hasShadow` Boolean (optional) - Whether window should have a shadow. This is only
     implemented on macOS. Default is `true`.
+  * `opacity` Double (optional) - Set the initial opacity of the window, between 0.0 (fully
```

Comment:
I used it here because `Double` is used elsewhere several times. https://github.com/electron/electron/search?l=Markdown&q=double&type=&utf8=%E2%9C%93

Reply:
Yeah, not saying all our docs exactly follow what should be going on (we have a lot of docs 😆 ), but news ones should follow the style guide so that our ts converter has an easier job